source/SIL.AppBuilder.Portal.Frontend/src/ui/components/header/display.tsx

```diff
@@ -48,23 +56,27 @@ class Header extends React.Component<Props & RouteComponentProps<{}>, State> {
+                  <span className='black-text font-lg'>
+                    FL
```

Comment:
We could add a function or helper to generate the name like

```
const initials = name =>
  name ?
    name
      .match(/\b(\w)/g)
      .join('')
      .substring(0, 2) : "";
```

Reply:
eventually. it'll be a part of hooking up to current user data.